src/components/SuiteHeader/SuiteHeader.story.jsx

```diff
@@ -92,6 +92,40 @@ const sideNavLinks = [
   },
 ];
 
+/* Sample of SuiteHeader usage with data hook
```

Comment:
This story failed jest with the "state change needs to be wrapped in act" error.  The code has value as a reference sample, so I left it in (commented out).  I'm happy to remove it if needed.